src/backend/routers/auth.py

```diff
@@ -133,17 +136,23 @@ async def oidc_authenticate(request: Request, session: DBSessionDep):
 
 
 @router.get("/logout", response_model=Logout)
-async def logout(request: Request):
+async def logout(
+    request: Request,
+    session: DBSessionDep,
+    token: dict | None = Depends(validate_authorization),
```

Comment:
what does having `Depends(validate_authorization)` as the default value here do? Also `depends` isn't imported here, is the line by accident?

Reply:
When you add any `Depends()` calls, whether during router definition or endpoint definition FastAPI will call those dependencies (in the order you added them) on that request. So here I'm adding the `validation_authorization` dependency. 

And nice catch, added the `Depends` back, think I forgot it during merge conflict